css/style.css

```diff
@@ -0,0 +1,582 @@
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: white;
+}
+
+.recent-works h4:not(.title) {
+  text-align: left;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: white;
+}
+
+.recent-works p {
+  font-family: 'Inter', sans-serif;
+  text-align: left;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 12px;
+  color: #091e42;
+}
+
+.tech-used {
+  height: 40px;
+  margin-bottom: 12px;
+}
+
+.recent-works .css {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 49px;
+  height: 40px;
+  border: 1px solid #8993a4;
+  font-family: Inter, sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  font-style: normal;
+  color: #091e42;
+  margin: 0 12px 0 0;
+  padding: 10px 12px;
+  box-sizing: border-box;
+}
+
+.recent-works .HTML {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 56px;
+  height: 40px;
+  border: 1px solid #8993a4;
+  list-style: none;
+  font-family: Inter, sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  font-style: normal;
+  color: #091e42;
+  margin: 0 12px 0 0;
+  padding: 10px 12px;
+  box-sizing: border-box;
+}
+
+.recent-works .boots {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 95px;
+  height: 40px;
+  border: 1px solid #8993a4;
+  list-style: none;
+  font-family: Inter, sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  font-style: normal;
+  color: #091e42;
+  margin: 0 12px 0 0;
+  padding: 10px 12px;
+  box-sizing: border-box;
+}
+
+.recent-works .ruby {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 61px;
+  height: 40px;
+  border: 1px solid #8993a4;
+  font-family: Inter, sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  font-style: normal;
+  color: #091e42;
+  margin: 0 12px 0 0;
+  padding: 10px 12px;
+  box-sizing: border-box;
+}
+
+button {
+  padding: 12px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  border: none;
+}
+
+form button {
+  padding: 12px;
+  width: 131px;
+  height: 48px;
+  align-self: center;
+}
+
+button:enabled {
+  color: #fff;
+  background-color: #f55800;
+}
+
+button:hover {
+  border: 1px solid #f55b00;
+  background-color: #fff5e1;
+  color: #f55800;
+}
+
+button:active {
+  background-color: #e05304;
+  border: 1px solid #f55b00;
+  color: #f4f1f0;
+}
+
+.recent-works button {
+  width: 124px;
+  height: 48px;
+  float: left;
+}
+
+/* First-Profesional-Art-Printing-Data */
+.cards-section {
+  padding-top: 30px;
+  display: grid;
+  gap: 20px;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  grid-template-rows: 1fr;
+}
```

Comment:
Please kindly make sure your design sticks as much as possible to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) 😉 😊

- [ ] I noticed that between your cards there is no space 🙃 
- [ ] The first button is in the second card but is supposed to be at the top of the first project 🙃 
- [ ] The width of the third button is not the same as the other buttons, please kindly fix it as well 😉 
- [ ] Also add the background image to your cards 😊 👍

Your design 

<img width="219" alt="Screen Shot 2022-09-21 at 16 24 21" src="https://user-images.githubusercontent.com/74501523/191536296-827f3e28-87ba-4b7f-93cc-7f75a69ef965.png">

Figma design

<img width="179" alt="Screen Shot 2022-09-21 at 16 25 44" src="https://user-images.githubusercontent.com/74501523/191536214-6a5a5ac5-6375-448a-8776-b6b68d773a06.png">